backend/src/graphql/enum/ContributionStatus.ts

```diff
@@ -0,0 +1,14 @@
+import { registerEnumType } from 'type-graphql'
+
+export enum ContributionStatus {
+  PENDING = 'PENDING',
+  DELETED = 'DELETED',
+  IN_PROGRESS = 'IN_PROGRESS',
```

Comment:
What the difference with PENDING? IN_PROGRESS is when their has been a question back?

Reply:
PENDING: A moderator has to check the contribution.
IN_PROGRESS: Moderator has questions (added a message) and is waiting for user response
Please resolve conversation when your question is answered completely